api/src/controllers/anime/update.action.ts

```diff
@@ -0,0 +1,39 @@
+import type { Request, Response, NextFunction } from '@@types/express.js';
+import { errors, entities } from '@@utils/index.js';
+import { Anime } from '@@entities/index.js';
+
+interface Params {
+    id: number
+};
+
+// Update
+/**
+ * Updates an Anime entity.
+ *
+ * @param req - The request object.
+ * @param res - The response object.
+ * @param next - The next function.
+ * @returns A JSON response with the updated Anime entity.
+ */
+async function update(req: Request, res: Response<Params>, next: NextFunction) {
```

Comment:
This will also have a `req.body` that needs to be validated, and typed out like you're doing in `create.action.ts`